Assets/StarterAssets/FirstPersonController/Scripts/FirstPersonController.cs

```diff
@@ -162,33 +163,34 @@ private void Move()
 
             if (sprinting)
             {
-                stamina -= 1 * Time.deltaTime;
+                stamina -= _staminaRechargeMultiplier * Time.deltaTime;
             } else if(stamina < maxStamina && !isFatigued)
             {
-                stamina += 1 * Time.deltaTime;
+                stamina += _staminaRechargeMultiplier * Time.deltaTime;
             }
 
             if (isFatigued)
             {
-                if(FatigueTimer+1f < Time.time)
+                if(FatigueTimer < Time.time)
                 {
-                    fatigueRecharge += 1 * Time.deltaTime;
+                    fatigueRecharge += _staminaRechargeMultiplier * Time.deltaTime;
                 }
                 if(fatigueRecharge >= fatigueTimer)
                 {
-                    //Debug.Log("Recharge complete");
+                    // Recharge complete
                     isFatigued = false;
-                    stamina = 0;
```

Comment:
Remove resetting of stamina to 0 when fatigueTimer is done since this approach allows stamina recharge during fatigue.